index.html

```diff
@@ -0,0 +1,235 @@
+                  </div>
+                </div>
+                <div class="col-8 col-md-12">
+                  <p  class="paragraph text-start text-md-center">
+                    The best start-ups of the year will be awarded after
+                    exhibits their technology
+                  </p>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div class="col-12 col-md-2 icon-left">
+              <div class="box-program align-items-center row py-3">
+                <div class="col-4 col-md-12">
+                  <div class="d-flex justify-content-around d-md-flex
+                    flex-md-column align-items-center gap-md-3">
+                    <img src="images/network.svg" class="icon2" alt="">
+                    <span class="fw-bold text-capitalize">Network</span>
+                  </div>
+                </div>
+                <div class="col-8 col-md-12">
+                  <p  class="paragraph text-start text-md-center">
+                    AfroTech is the place to growth you network and coonect with
+                    others.
+                  </p>
+                </div>
+
+              </div>
+
+            </div>
+            <div class="col-12 col-md-2 icon-left">
+              <div class="box-program align-items-center row py-3">
+                <div class="col-4 col-md-12">
+                  <div class="d-flex justify-content-around d-md-flex
+                    flex-md-column align-items-center gap-md-3">
+                    <img src="images/job.svg" class="icon2" alt="forum-image">
+                    <span class="fw-bold text-capitalize"> Job</span>
+                  </div>
+                </div>
+                <div class="col-8 col-md-12 ss">
+                  <p  class="paragraph text-start text-md-center">
+                    Are you a jobseeker or a recruiter, we provide the
+                    right tools for you to success.
+                  </p>
+                </div>
+
+              </div>
+
+            </div>
+
+          </div>
+
+          <button class="btn-first btn my-3 py-3 px-5 mt-4"> Join AfroTech expo</button>
+          <a class="link text-center"> See the whole program</a>
+        </div>
+      </section>
+
+      <!--SECTION SPEAKER *************************************** -->
+      <section class="speakers my-5 text-center px-5 container">
+        <h2> Featured Speakers</h2>
+
+        <div class="line my-3"> </div>
+        <!-- Box speaker-box-->
+        <div class="grid-speaker" id="grid-speaker">
+
+        </div>
+        <!-- <div class="arrow-down mt-5">
+          <span>more</span>
+          <svg xmlns="http://www.w3.org/2000/svg" aria-hidden="true" role="img"
+            width="1em" height="1em" preserveAspectRatio="xMidYMid meet"
+            viewBox="0 0 20 20"><path fill="currentColor" d="m5 6l5 5l5-5l2 1l-7
+              7l-7-7z" /></svg>
+          </div> -->
```

Comment:
- [ ] Kindly remove all unused or unwanted codes from your file. This helps to keep your codebase clean. 👍